src/main/scala/codecheck/github/models/Repository.scala

```diff
@@ -66,9 +66,12 @@ case class Repository(value: JValue) extends AbstractJson(value) {
   def name = get("name")
   def full_name = get("full_name")
   def url = get("url")
+  def language = get("language")
+  def stargazers_count = get("stargazers_count").toLong
 
   def description = opt("description")
-  def open_issues_count = get("open_issues_count").toInt
+  def open_issues_count = get("open_issues_count").toLong
```

Comment:
This was the only compatibility change in this pull request.

There was no test case to catch it.  The `RepositoryCommand` uses it, but the generic `PrintList.build` function doesn't care about the type.